Crash consistent VM restore points (preview)/createRestorePoint.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "RPCName": {
+      "type": "string",
+      "defaultValue": "<RestorePointCollectionName>"
+    },
+    "RPCResourceId": {
+      "defaultValue": "/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroupName>/providers/Microsoft.Compute/restorePointCollections/<RestorePointCollectionName>",
+      "type": "String"
+    }
```

Comment:
Remove?

Reply:
@prchin, I have removed it. Can you check again if the tempalte is correct?